README.md

```diff
@@ -1,2 +1,401 @@
+        example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bash
+      ./build/example-application/zephyr/projects/zxa_example_project/native_posix/zephyr/zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister \
+        -T example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister --coverage \
+        -T example-application/zephyr
+      ```
+
+### macOS
+
+1. Build projects directly from the command line
+
+      ```bash
+      cd ~/zxa
+      west build -p auto -b native_posix \
+        example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bash
+      ./build/example-application/zephyr/projects/zxa_example_project/native_posix/zephyr/zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister \
+        -T example-application/zephyr/projects/zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bash
+      zephyr/scripts/twister --coverage \
+        -T example-application/zephyr
+      ```
+
+### Windows
+
+1. Build projects directly from the command line
+
+      ```bat
+      cd %HOMEPATH%\zxa
+      west build -p auto -b native_posix \
+        example-application\zephyr\projects\zxa_example_project
+      ```
+
+1. Run projects from the command line
+
+      ```bat
+      .\build\example-application\zephyr\projects\zxa_example_project\native_posix\zephyr\zephyr.exe
+      ```
+
+1. Build and run projects using Zephyr's `twister script`
+
+      ```bat
+      zephyr\scripts\twister \
+        -T example-application\zephyr\projects\zxa_example_project
+      ```
+
+1. Build and run the all projects (including projects and tests)
+   with code coverage using Zephyr's `twister script`
+
+      ```bat
+      zephyr\scripts\twister --coverage \
+        -T example-application\zephyr
+      ```
+
+
+# Module Contribution Guidelines
+
+## Design Principles
+
+The ZXA module is designed around the following principles:
+
+- Keep all Zephyr-specific content within the (required) `zephyr` folder.
+
+  - This enables adding the example on to an existing repository.
+
+- Follow the Zephyr repository structure, mechanisms, and rules
+  whenever possible.
+
+  - This minimizes the variance from what users and developers expect,
+    simplifies maintenance, and reuses existing solutions.
+
+- Extend the Zephyr repository structure to support an arbitrary number
+  of applications and project configurations.
+
+  - The `projects` sub-tree identifies where non-test, non-sample projects
```

Comment:
I would remove this section altogether

Reply:
This is setting up a scalable organization that will grow more very shortly.  I see no benefit in simplifying this for such a short window of time.

Reply:
I agree with the premise that in the future we will have multiple applications, either in this repo or in an additional one. But not in the first pass.

Reply:
If we only have one, then it needs to be the rich one to verify the build system and be part of the LTS.  The simplest solution is make two now.